src/hubmap_translation/addl_index_transformations/portal/tests/test_add_partonomy.py

```diff
@@ -1,20 +1,31 @@
 import pytest
 
 from hubmap_translation.addl_index_transformations.portal.add_partonomy import _get_organ_iri
+from hubmap_translation.addl_index_transformations.portal.translate import TranslationException
 
 
 @pytest.mark.parametrize(
     "doc, expected_organ_iri",
     [
         pytest.param(
-            {"origin_samples": [{"organ": "UT"}]}, None, id="valid organ"
+            {}, None, id="empty doc does not throw exception"
+        ),
+        pytest.param(
+            {"uuid": "test_dataset_uuid", "entity_type": "Donor"}, None, id="Donor is expected to have missing origin_samples"
```

Comment:
```suggestion
            {"uuid": "test_uuid", "entity_type": "Donor"}, None, id="Donor is expected to have missing origin_samples"
```

Small idea to make this a bit clearer at a glance 👍🏻